postman/Fhir.postman_collection.json

```diff
@@ -124,11 +172,46 @@
 					"name": "PUT Binary PDF",
 					"request": {
 						"auth": {
-							"type": "bearer",
-							"bearer": [
+							"type": "oauth2",
+							"oauth2": [
+								{
+									"key": "redirect_uri",
+									"value": "http://localhost",
+									"type": "string"
+								},
+								{
+									"key": "state",
+									"value": "abc",
+									"type": "string"
+								},
+								{
+									"key": "scope",
+									"value": "profile openid",
+									"type": "string"
+								},
+								{
+									"key": "clientId",
+									"value": "{{CLIENT_ID}}",
```

Comment:
We should add to the docs how to set the value for the new variable  `CLIENT_ID`

I couldn't find it on our docs. Maybe I missed it. Before we were using the `COGNITO_TOKEN` variable

Reply:
updated